webapp/src/utils/link_only_renderer.js

```diff
@@ -0,0 +1,27 @@
+import RemoveMarkdown from './remove_markdown';
+
+function getScheme(url) {
+    const match = (/([a-z0-9+.-]+):/i).exec(url);
```

Comment:
perhaps be explicit for http(s) scheme?
`/^(http:\/\/|https:\/\/)/`

Reply:
Sorry, I need a little more time to understand here. I'll comment later.

Reply:
Practically, `http` and `https` are enough here. However, Mattermost always create link for `ftp`, `tel` and `mailto`, and we can set arbitrary url schemes. 
https://docs.mattermost.com/administration/config-settings.html#custom-url-schemes

I think we should follow this specs here, so I want to leave this for now.
As you may know, existing code doesn't follow that specs and some bugs remain, so I'll fix this as another PR.

Reply:
Sounds good to me.